src/index.html

```diff
@@ -6,9 +6,66 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+  <header class="header">
+    <a class="img" href="http://localhost:8080/logo.6023b87e.png"><img  src="./images/logo.png" alt="logo"></a>
+
+    <nav class="nav">
+      <button class="button"></button>
+      <ul class="nav__list">
+        <li>
+          <a class="nav__link is-active" href="http://localhost:8080/">
+            Apple
+          </a>
+        </li>
```

Comment:
Need to fix it everywhere
```suggestion
        <li class="nav__item">
          <a class="nav__link is-active" href="#apple">
            Apple
          </a>
        </li>
```

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3981/files#r1361546396
not fixed